helmfile/01-prometheus-operator.yaml

```diff
@@ -27,8 +27,7 @@ releases:
   chart: upstream/kube-prometheus-stack
   version: 12.8.0
   missingFileHandler: Error
-  needs:
-  - cert-manager/cert-manager
+  disableValidationOnInstall: true
```

Comment:
Maybe worth adding a comment with the rational here? It would be nice to keep track of if this was fixed upstream at some point so that you could still keep validation with CRD-installing charts.